docs/java/index.md

```diff
@@ -57,9 +57,26 @@ The SDK contains the following tools:
 
 ## Workshops
 
-- [Intro to Temporal Java SDK](https://www.youtube.com/watch?v=VoSiIwkvuX0)
+- [Java SDK Workshop #1](https://www.youtube.com/watch?v=VoSiIwkvuX0)
+
+  - Intro
+  - Workflows
+  - Activities
+  - Testing
 
 - [Java SDK Workshop #2](https://www.youtube.com/watch?v=h-TSDMULCf0)
+  - Client APIs
+  - Sleeping Durations (cron, periodic execution, time)
+  - Versioning
+  - Error Handling
+  - Dynamic Workflows / Activities
+- [Java SDK Workshop #3](https://www.youtube.com/watch?v=8DFox0fGjzI)
+  - Workflow and Activity Types, Signal and Query Names
+  - Different types of Stubs
+  - Dynamic Signal/Query Handlers
+  - ContinueAsNew and Signals
+  - Client Error Handling
+  - Setting up client SSL/mTLS
```

Comment:
I think we should let YouTube do what YouTube is good at, and just have a general "Java SDK playlist" link -> https://www.youtube.com/playlist?list=PLl9kRkvFJrlQ8KsM6m9cFfCeQegq_B8x4